docs/content/ecs/ecs-system-update-order.md

```diff
@@ -30,12 +30,12 @@ The groups are executed in the following order:
 * `SpatialOSUpdateGroup` - Most systems which use SpatialOS components should run in this group.
 * `SpatialOSSendGroup` - This group contains all systems related to sending data to SpatialOS .
   * `InternalSpatialOSSendGroup` - This group is used by the `SpatialOSSendSystem` to handle sending all replicated components to SpatialOS. **(Internal use only)**
-  * `CustomSpatialOSSendGroup` -  This group contains all systems running custom replication logic. If you create a [custom replication system](./custom-replication-system.md), add your system to this group.
-  * `InternalSpatialOSCleanGroup` - This group is used by the `CleanReactiveComponentsSystem` to remove all [reactive](./reactive-components.md) and [temporary components](./temporary-components.md) at the end of each update loop. **(Internal use only)**
+  * `CustomSpatialOSSendGroup` -  This group contains all systems running custom replication logic. If you create a [custom replication system]({{urlRoot}}/content/ecs/custom-replication-system.md), add your system to this group.
+  * `InternalSpatialOSCleanGroup` - This group is used by the `CleanReactiveComponentsSystem` to remove all [reactive]({{urlRoot}}/content/ecs/reactive-components.md) and [temporary components]({{urlRoot}}/content/ecs/temporary-components.md) at the end of each update loop. **(Internal use only)**
 
 Here's a diagram of the update order:
 
-![Update order](../../assets/update-order.png)
+![Update order]({{assetRoot}}assets/update-order.png)
```

Comment:
Could this URL be missing a "/"?

Reply:
It appears to render okay - I think `assetRoot` must have a trailing /

Reply:
AssetRoot requires `assets/` with no preceding /. So this is correct.